Samples/UniversalDecoder/.gitignore

```diff
@@ -0,0 +1,59 @@
+# Logs
```

Comment:
Maybe we can get rid of this file and move to the root .gitignore.

Reply:
removed